src/UI/Panels/FreeCamPanel.cs

```diff
@@ -685,7 +685,7 @@ internal void Update()
                     FreeCamPanel.EndFreecam();
                     return;
                 }
-
+                FreeCamPanel.connector.ShouldMoveToOriginalPosition(FreeCamPanel.ourCamera);
```

Comment:
```suggestion
                FreeCamPanel.connector?.ShouldMoveToOriginalPosition(FreeCamPanel.ourCamera);
```

Reply:
To reduce the calls made from the FreeCamPanel to the UnityIGCSConnector, wouldn't it make more sense to execute this at the start of `ExecuteCameraCommand`?

Reply:
You're right, there's no reason to call it within FreeCamPanel, oversight from my side.